lib/UnoCore/Targets/iOS/build.sh

```diff
@@ -5,6 +5,22 @@ set -e
 set -o pipefail
 cd "`dirname "$0"`"
 
+if ! which xcodebuild > /dev/null 2>&1; then
+    echo "ERROR: Unable to find the 'xcodebuild' command." >&2
+    echo -e "\nYou can download Xcode Command Line Tools from this page:" >&2
+    echo -e "\n    https://developer.apple.com/xcode/\n" >&2
+    exit 1
+fi
```

Comment:
This one looks very familiar ;)

Is there any reasonable way we could define this as a function in some helper-source rather than repeating it? If not, twice isn't *too* bad, I just don't want us to have to remember to update two code-sites if a link changes or something...

Reply:
One is for iOS and the other for macOS. It is difficult to share this code between the two, so in this case I think code duplication is better than a bunch of engineering.